deltacat/compute/compactor_v2/utils/exception_handler.py

```diff
@@ -0,0 +1,138 @@
+import botocore
```

Comment:
This exception handler can affect all of deltacat. Can it be in `exceptions` module at the root? Do we want to create different handlers for different components?